src/index.js

```diff
@@ -129,13 +135,15 @@ function getTabindex(node) {
     return 0;
   }
 
-  // in Chrome, <audio controls/> and <video controls/> elements get a default
+  // in Chrome, <details/>, <audio controls/> and <video controls/> elements get a default
   //  `tabIndex` of -1 when the 'tabindex' attribute isn't specified in the DOM,
   //  yet they are still part of the regular tab order; in FF, they get a default
   //  `tabIndex` of 0; since Chrome still puts those elements in the regular tab
-  //  order, consider their tab index to be 0
+  //  order, consider their tab index to be 0.
   if (
-    (node.nodeName === 'AUDIO' || node.nodeName === 'VIDEO') &&
+    (node.nodeName === 'AUDIO' ||
+      node.nodeName === 'VIDEO' ||
+      node.nodeName === 'DETAILS') &&
```

Comment:
Nice catch! I see that the `<summary>` element gets `tabIndex === 0` so we're good there too with line 129 above.